tasks/main.yml

```diff
@@ -30,6 +54,55 @@
     state: directory
   notify: Restart uchiwa
 
+- block:
+  - name: Create directory for publickey
+    file:
+      path: "{{ uchiwa_publickey_path | dirname }}"
+      owner: "{{ uchiwa_user }}"
+      group: "{{ uchiwa_group }}"
+      state: directory
+      mode: 0755
+
+  - name: Create publickey
+    copy:
+      dest: "{{ uchiwa_publickey_path }}"
+      mode: "{{ uchiwa_publickey_mode }}"
+      owner: "{{ uchiwa_user }}"
+      group: "{{ uchiwa_group }}"
+      content: "{{ uchiwa_publickey }}"
+    notify: Restart uchiwa
+  when:
+    - "'uchiwa' in uchiwa_config"
```

Comment:
these four conditions are always met because of the assert task above and not necessary.